packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/pages/edit/page-list.astro

```diff
@@ -0,0 +1,121 @@
+---
+// @ts-expect-error - Some types can only be imported from the Astro runtime
+import { SiteConfig, db, eq, asc, PageData } from 'astro:db';
+import { Layout, FormattedDate } from 'studiocms-dashboard:components';
+import Config from 'virtual:studiocms/config';
+import { authHelper, pluginList, type Locals } from 'studiocms:helpers';
+import { StudioCMSRoutes, getEditRoute, getDeleteRoute } from 'studiocms-dashboard:routeMap';
+import type { SiteConfigResponse, pageDataReponse } from 'studiocms:components';
+
+const { dashboardConfig: { developerConfig: { testingAndDemoMode } } } = Config;
+
+const { 
+    authLinks: { 
+        loginURL 
+    }, 
+    mainLinks: { 
+        userProfile,
+        dashboardIndex,
+        pageEdit,
+    } 
+} = StudioCMSRoutes;
+
+const contextConfig: SiteConfigResponse = await db.select().from(SiteConfig).where(eq(SiteConfig.id, 1)).get();
```

Comment:
Same this as above with this `1`, probably wants to be named

Reply:
`1`, is the arbitrary id number that i assigned to the first entry to the SiteConfig Table in astroDB....  its also SUPPOSED to be the only entry in said table. and is created by the Database Setup page

Reply:
Done as of [3810634](https://github.com/astrolicious/studiocms/pull/77/commits/3810634c3e7fd7dd60ac90ef1dda6b1115e7e026)